lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1351,11 +1354,17 @@ slotParams gp =
         (gp ^. #getGenesisBlockDate)
         (gp ^. #getActiveSlotCoefficient)
 
+-- | Protocol parameters that can be changed through the update system.
+--
 data ProtocolParameters = ProtocolParameters
     { decentralizationLevel
         :: Quantity "percent" Percentage
+        -- ^ The current level of decentralization in the network.
+        --   * '  0%' indicates that the network is /completely federalized/.
+        --   * '100%' indicates that the network is /completely decentralized/.
     , txParameters
         :: TxParameters
+        -- ^ Parameters relating to transactions.
```

Comment:
```suggestion
        -- ^ Parameters which affect transaction creation.
```

Reply:
Fixed in cfaf9ac6a561b94034465d28ad883835a13ac38b.